source/windows/windows_pki_utils.c

```diff
@@ -88,11 +88,15 @@ int aws_load_cert_from_system_cert_store(const char *cert_path, HCERTSTORE *cert
         CERT_STORE_PROV_SYSTEM_A, 0, (HCRYPTPROV)NULL, CERT_STORE_OPEN_EXISTING_FLAG | store_val, store_path);
 
     if (!*cert_store) {
+        WCHAR wszMsgBuff[512];
+        FormatMessage(
+            FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_IGNORE_INSERTS, NULL, GetLastError(), 0, wszMsgBuff, 512, NULL);
```

Comment:
You should probably store off the value of GetLastError() like:

`int win_error = GetLastError();`

 Each time you call any Windows function there's a  chance GetLastError()'s value may change. So there's a chance that the call to FormatMessage() will change its value before you call GetLastError() again when logging a few lines below